server/src/main/java/org/tctalent/server/service/db/UserService.java

```diff
@@ -76,6 +80,14 @@ public interface UserService {
     void updatePassword(UpdateUserPasswordRequest request);
     void updateUserPassword(long id, UpdateUserPasswordRequest request);
 
+
+    // Generates an email verification token and sends a verification email.
```

Comment:
You should used standard JavaDoc comments like elsewhere in the code.
For example.

```suggestion
    /**
     * Generates an email verification token and sends a verification email.
     */
```

Reply:
I'll make sure to fix this and use standard JavaDoc comments, Thank you!